src/Layouts/Navbar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { CgBmw } from 'react-icons/cg';
+import { AiOutlineHeart, AiOutlineShoppingCart } from 'react-icons/ai';
+
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+function Navbar({ title }) {
+  return (
+    <nav className='navbar shadow-lg bg-neutral text-neutral-content '>
+      <div className='container mx-auto'>
+        <div className='flex-none px-2 mx-2'>
+          <CgBmw className='inline pr-2 text-3xl' />
+          <Link to='/' className='text-lg font-bold align-middle'>
+            {title}
+          </Link>
+        </div>
+
+        <div className='flex-1 px-2 mx-2'>
+          <div className='flex justify-end'>
+            <form className='flex '>
+              <input className='rounded' type='text' placeholder='Search' />
```

Comment:
adding padding inside the search bar might look nicer, the placeholder and the searchbar border is pretty mashed up. 
padding left 1rem or overall padding ( as long as it doesnt ruin the navbar itself) might help as well.